utils/reviewsUtils.js

```diff
@@ -50,3 +51,36 @@ export async function postReview(review) {
     })
     return res.json();
 }
+
+async function getMovieReview(id) {
+  const res = await fetch(`${API_BASE}/reviews?filters[movie]=${id}`);
+  const payload = await res.json();
+  const modifiedArr = payload.data.map((obj) => ({
+    id: obj.id,
+    ...obj.attributes
+  }))
+  return modifiedArr;
+}
+
+export async function getAverageRating(id) {
+  const reviewsList = await getMovieReview(id);
+  const imdbRes =  await imdbRating(id);
+  let averageRating, maxRating;
+  if (reviewsList.length >= 5) {
+    let sumRatings = 0;
+    reviewsList.forEach((review) => {
+      sumRatings += review.rating;
+    }
+    )
+    averageRating = sumRatings / reviewsList.length;
+    maxRating = 5;
+  }else {
+    averageRating = imdbRes;
+    maxRating = 10;
```

Comment:
I think it would have been better to normalize the rating to always be 0-5, even if it's loaded from IMDB. You could have done that here by just setting `averageRating = imdbRes / 2`.